packages/dapp/src/registry/RegistryWrapper.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
```

Comment:
Nothing changed here, whole file is made up of stuff cut from `RegistrySection.tsx`.